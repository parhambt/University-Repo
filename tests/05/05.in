reserve_table 1 3
show_table_info 3
enter 2
reserve_table 12 1
switch 1
show_table_info 3
exit 1
show_table_info 1
