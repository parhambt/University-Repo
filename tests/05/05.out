Paria sits at table 3
Table ID: 3
People at the table: Paria
Table remaining capacity: 1
Waiting queue length: 0
Table 1: 4 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 1 0
Table 4: 2 0
Bob sits at table 1
Paria switches seats with Bob!
Table ID: 3
People at the table: Bob
Table remaining capacity: 1
Waiting queue length: 0
Paria exits!
Table ID: 1
People at the table:
Table remaining capacity: 4
Waiting queue length: 0